docs(cloud): add "For Designers" section to Widgetbook Cloud docs (#2000)
widgetbook/widgetbook: docs/cloud/for-designers/overview.mdx

# For Designers

> Widgetbook Cloud has two audiences: the developers who build the UI, and the designers who own how it should look. Most of these docs are written for developers. This section is for you. It assumes you have never opened a Flutter project and never will.

Reviewer: Let's not make assumptions about the role of the reader and instead address the role directly. 

```suggestion
Widgetbook Cloud has two audiences: the developers who build the UI, and the designers who own how it should look. Most of these docs are written for developers. This section is for Designers, which usually never open Flutter projects and likely never will.
```

Reviewer: Applies generally for the page and likely the whole PR.

You do not need to run code, check out a branch, or install anything. Every pull request produces a [review](/cloud/reviews) you open in your browser. You compare the implementation against your design and leave feedback the developer sees on the change itself.

<Image theme="light" src="/assets/cloud/review.light.png" />
<Image theme="dark" src="/assets/cloud/review.dark.png" />

<Info>
  New to the words developers use around this workflow? Start with the

>   [Glossary](/cloud/for-designers/glossary). The vocabulary is usually what causes

Reviewer: Let's remove fluff text like 
> The vocabulary is usually what causes...

Personally I think this whole `<Info>` section can be removed.

  the confusion, not the tool.
</Info>

## The two-stage review

A single UI review has two passes. They are usually done by two different people.

> 1. **Filter out the unintended.** The developer who opened the pull request looks first, for changes they did not mean to make. A component that shifted, spacing that moved, a state that now renders differently. They fix those before asking anyone else to look.

Reviewer: ```suggestion
1. **Filter out the unintended.** The developer who opened the pull request looks for changes they did not mean to make. A component that shifted, spacing that moved, a state that now renders differently. They fix those before asking other team members to review.
```

> 2. **Judge whether it is right.** Once only intentional changes remain, the next reviewer decides whether those changes are correct. When that is a design question, that reviewer is you.

Reviewer: What is "it"? Generally, we should refrain from using filling words like it, they, that, etc and instead use describing nouns as it's easier to follow the content.

The first pass is a filter. A review that reaches you should contain only deliberate changes. Your job is not to hunt for accidents the developer already caught. It is to judge whether the deliberate changes are right.

> ## In this section

Reviewer: The headline seems out of place and can be improved.

<CardGroup>

>   <Card title="When to involve a designer" icon="user-plus" href="/cloud/for-designers/when-to-involve-a-designer">

Reviewer: Using a card group as an overview is a good idea, but the actions should be mentioned in the existing pattern. 

E.g. see the Reviewers Actions:
- Request a reviewer (instead of doing a review)
- Submit a review
- etc

Also note that we write those action in an imperative mood.

Generally, I think "work with Figma" sounds very generic. I like to have specific actions as docs pages as that's easiest to browse as long as the list of mentioned actions is small. 
Note that we also have 3 Pages (for reviewing, requesting reviewers and submitting) resolving around managing reviews. Each of the pages is small.
